nautobot/dcim/forms.py

```diff
@@ -1797,9 +1797,9 @@ def save(self, *args, **kwargs):
         # call IPAddress setter method to deconstruct netaddr.IPNetwork
         # into host, broadcast, and prefix_length model fields
```

Comment:
need to update this comment. it's also possible that we don't need this method at all if we're not deconstructing anything

Reply:
Agreed. Also makes me wonder if we even need the `.save()` method overload now, as this appears to be doing the right thing. Since the form would already be populating this field from `cleaned_data`. ICYMI: I modified the behavior of the fields somewhat recently in #246. Is this my fault? 🙈